### ISSUE TITLE
Navigation error in admin 

### ISSUE THREAD

**Issue by neosin** [loic-sharma/Anvil#6]
Install worked great.
Went to admin and clicked around.
Clicked Navigation (http://127.0.0.1/admin/navigation)
and got this error, no idea why

ErrorException
…\classes\Anvil\Plugins\NavigationPlugin.php : 52
Illuminate\Exception\Handler handleError
…\classes\Anvil\Plugins\NavigationPlugin.php : 52
Anvil\Plugins\NavigationPlugin groups
…\storage\views\59ffd719d525ebfbc8214e25c4626940 : 12 

ErrorException
Undefined property: Anvil\Plugins\NavigationPlugin::$model

```
    /**
* Get all of the navigation groups.
*
* @return array
*/
public function groups()
{
return $this->model->all();
}
```

**Comment by loic-sharma** [comment]
Yep, I know. The admin panel is still pretty unstable. I changed a good chunk of the underlying code but didn't update the admin panel, so some parts are still broken. Anvil is still very much still a work in progress! :P
